.jazzy.yml

```diff
@@ -0,0 +1,50 @@
+# jazzy --swift-version 2.2 -m Deferred -g "https://github.com/bignerdranch/Deferred" -a "Big Nerd Ranch" -u "http://bignerdranch.github.io" --module-version=2.0.0 -r "http://bignerdranch.github.io/Deferred/"
+author: "Big Nerd Ranch"
+author_url: "http://bignerdranch.github.io"
+copyright: "© 2014-2016 [Big Nerd Ranch](https://www.bignerdranch.com) under [MIT license](https://github.com/bignerdranch/Deferred/blob/master/LICENSE.txt)."
```

Comment:
It does. The answer you wanted to read is:
http://meta.stackexchange.com/a/121321 The asker wasn't writing a valid
URL; the scheme-relative ones begin with a leading //, not directly with
the hostname.

I'm using it already on several GitHub READMEs, so. :) To see it in action,
check the raw view here:
https://github.com/bignerdranch/jws-coffeerun/blob/master/README.md

--
Jeremy W. Sherman <jeremy@bignerdranch.com>

2016-04-12 13:41 GMT-04:00 Zachary Waldowski <notifications@github.com>:

> In .jazzy.yml
> <https://github.com/bignerdranch/Deferred/pull/94#discussion_r59419682>:
>
> > @@ -0,0 +1,50 @@
> > +# jazzy --swift-version 2.2 -m Deferred -g "https://github.com/bignerdranch/Deferred" -a "Big Nerd Ranch" -u "http://bignerdranch.github.io" --module-version=2.0.0 -r "http://bignerdranch.github.io/Deferred/"
> > +author: "Big Nerd Ranch"
> > +author_url: "http://bignerdranch.github.io"
> > +copyright: "© 2014-2016 [Big Nerd Ranch](https://www.bignerdranch.com) under [MIT license](https://github.com/bignerdranch/Deferred/blob/master/LICENSE.txt)."
>
> Doesn't seem like it
> http://meta.stackexchange.com/questions/121293/markdown-support-for-urls-without-a-protocol
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/bignerdranch/Deferred/pull/94/files/58b015f914eaaea5560f6ec8ea2a5f448bbcb355#r59419682>
>